AlphaWallet/Tokens/Types/Oneinch/Oneinch.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  OneinchHolder.swift
+//  AlphaWallet
+//
+//  Created by Vladyslav Shepitko on 11.11.2020.
+//
+
+import Foundation
+import PromiseKit
+import Moya
+
+class Oneinch {
+    private(set) static var availableTokens: [Oneinch.ERC20Token] = []
```

Comment:
Can we make this be instance-side?

Reply:
@hboon not clearly understand about this, can u provide more details?

Reply:
Remove `static` :)

```
private(set) var availableTokens: [Oneinch.ERC20Token] = []
```